x-pack/plugins/infra/public/store/local/metric_time/epic.ts

```diff
@@ -3,6 +3,7 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
```

Comment:
I guess this is just some white space weirdness where your IDE Is adding spaces? ;-)

Reply:
I am sure it is my ODC and add a line